setup_cfg_fmt.py

```diff
@@ -369,8 +387,8 @@ def format_file(
     # normalize names to underscores so sdist / wheel have the same prefix
     cfg['metadata']['name'] = cfg['metadata']['name'].replace('-', '_')
 
-    # if README.md exists, set `long_description` + content type
-    readme = _first_file(filename, 'readme')
```

Comment:
if we're no longer using this function it doesn't need to exist -- it was meant to be reused for all the usecases

I would prefer that the function is kept since it is useful for licenses and other file detection